### PR TITLE
ddb interface changes

### PR DESCRIPTION
**Pull request by petejan**
clean up csv vs ddb query selection into a function,
Also fix missing database field because the last column was cut off

### CODE REVIEW

**Comment by ggalibert** [comment]
Sounds good, will review and include this in the next release.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+data
+temp
```
Comment (ggalibert) [inline .gitignore]: I won't include this file. Better remove it from the commit.

File: Java/src/org/imos/ddb/Test.java [hunk 2]
```diff
@@ -72,8 +72,10 @@ public static void main(String[] args) {
 //		String mdbFile = "/home/ggalibert/Documents/IMOS_toolbox/data_files_examples/AIMS/Paul_Rigby/OceanDB.mdb";
 //		String connection = "jdbc:ucanaccess://" + mdbFile + ";jackcessOpener=org.imos.ddb.CryptCodecOpener;SingleConnection=true";
 //		String connection = "jdbc:ucanaccess://" + mdbFile + ";jackcessOpener=org.imos.ddb.CryptCodecOpener";
-		String connection = "jdbc:postgresql://localhost/Darren_2016-03-24";
-		String user = "myUser";
+//		String connection = "jdbc:postgresql://localhost/Darren_2016-03-24";
+		String connection = "jdbc:postgresql://localhost/IMOS-DEPLOY";
+//		String user = "myUser";
+		String user = "csiro";
```
Comment (ggalibert) [inline Java/src/org/imos/ddb/Test.java]: Won't include changes on this file. Better remove it from the commit.

File: NetCDF/makeNetCDFCompliant.m [hunk 3]
```diff
@@ -226,15 +214,17 @@
...
+            Sensors = executeQuery('Sensors', 'SensorID',   InstrumentSensorConfig(i).SensorID);
             if ~isempty(Sensors)
                 % check if this sensor is associated to the current IMOS parameter
                 if isfield(Sensors, 'Parameter')
-                    parameters = textscan(Sensors.Parameter, '%s', 'Delimiter', ',');
-                    if ~isempty(parameters)
-                        parameters = parameters{1};
-                        if any(strcmpi(IMOSParam, parameters))
-                            target = Sensors.SerialNumber;
+                    if ~isempty(Sensor.Parameters)
```
Comment (ggalibert) [inline NetCDF/makeNetCDFCompliant.m]: Typo. Should be:
```Matlab
if ~isempty(Sensors.Parameter)
```

**Comment by petejan** [comment]
Yes sorry about all the reverts, pressed the wrong button (twice).

Fixed those comments in the branch, does that work?

**Comment by ggalibert** [comment]
Will include these changes in the next release. Thanks.

**Comment by ggalibert** [comment]
Included in v2.5.32 via commits:
https://github.com/aodn/imos-toolbox/commit/a6161b1af5a805bf15bf19e70e92882365bcdcf3
https://github.com/aodn/imos-toolbox/commit/859b5842ce9ce401c65fd0ac90b8b33c19d6bd90
https://github.com/aodn/imos-toolbox/commit/4daebca287bc4b257cadefe1d9426e1dc0793932

**Pull request closed without merging**